src/client/datascience/kernel-launcher/localPythonAndRelatedNonPythonKernelSpecFinder.ts

```diff
@@ -89,9 +89,6 @@ export class LocalPythonAndRelatedNonPythonKernelSpecFinder extends LocalKernelS
         );
     }
     /**
-     * If user has python extension installed, then we'll not list any of the globally registered Python kernels.
```

Comment:
This line here (which already exists) seemed like the important check:
https://github.com/microsoft/vscode-jupyter/blob/3e6f5860a238b2a340efd317649d6d8ec3f0129e/src/client/datascience/kernel-launcher/localPythonAndRelatedNonPythonKernelSpecFinder.ts#L137

If we can't match up an interpreter then we do hide the global specs, but if we are able to match up the interpreter then the is no reason to hide.